xbmc/interfaces/legacy/Dialog.h

```diff
@@ -280,16 +280,17 @@ namespace XBMCAddon
 #ifdef DOXYGEN_SHOULD_USE_THIS
       ///
       /// \ingroup python_Dialog
-      /// \python_func{ xbmcgui.Dialog().textviewer(heading, text) }
+      /// \python_func{ xbmcgui.Dialog().textviewer(heading, text, usemono) }
       ///------------------------------------------------------------------------
       ///
-      /// **TextViewe dialog**
+      /// **TextViewer dialog**
       ///
       /// The text viewer dialog can be used to display descriptions, help texts
       /// or other larger texts.
       ///
       /// @param heading       string or unicode - dialog heading.
       /// @param text          string or unicode - text.
+      /// @param usemono       [opt] bool - use monospace font
       ///
```

Comment:
Could you add this few lines down as changelog?
`@python_v18 New optional param added **usemono**. `

Reply:
done.